### PR TITLE
COM-3260 Added fulfillment delivery type in cart

### PR DESCRIPTION
**Pull request by OnkarDPatil**

### CODE REVIEW

File: scripts/modules/cart-common-view.js [hunk 1]
```diff
@@ -226,20 +226,33 @@ define(['modules/api',
...
+                    cartItem.set('fulfillmentLocationCode', oldLocationCode);
+                  }
+                );
+              } else if (value == 'Pickup') {
+                //first we get the correct product code for this item.
+                //If the product is a variation, we want to pass that when searching for inventory.
+                cartItem.set('fulfillmentMethod', value);
+                cartItem.set('fulfillmentLocationName', '');
+                cartItem.set('fulfillmentLocationCode', '');
+                var productCode =
```
Comment (kibo-srikanthbandlamudi) [inline scripts/modules/cart-common-view.js]: Call this pickupProductCode for uniformity

**Review by kibo-srikanthbandlamudi — CHANGES_REQUESTED** [review]

File: scripts/modules/cart-common-view.js (hunk 1 above)
Comment (OnkarDPatil) [reply scripts/modules/cart-common-view.js]: Done

File: labels/de-DE.json [hunk 2]
```diff
@@ -679,5 +679,6 @@
   "yes": "Ja",
   "yourAccount": "Ihr Konto",
   "yourOrderNumber": "Ihre Bestellnummer lautet <strong> {0} </ strong>.",
-  "memberProducts": "Produits des membres(1)"
+  "memberProducts": "Produits des membres(1)",
```
Comment (Rameshdesaiise) [inline labels/de-DE.json]: Arrange these keys Alphabetically.

File: labels/en-US.json [hunk 3]
```diff
@@ -839,5 +839,6 @@
   "yes": "Yes",
   "yourAccount": "Your Account",
   "yourOrderNumber": "Your order number is <strong>{0}</strong>.",
-  "memberProducts": "Member Products"
+  "memberProducts": "Member Products",
```
Comment (Rameshdesaiise) [inline labels/en-US.json]: Arrange these keys Alphabetically.

**Review by Rameshdesaiise — CHANGES_REQUESTED** [review]
added some review comments, please fix those

File: labels/de-DE.json (hunk 2 above)
Comment (OnkarDPatil) [reply labels/de-DE.json]: Done

File: labels/en-US.json (hunk 3 above)
Comment (OnkarDPatil) [reply labels/en-US.json]: Done

**Pull request merged**